src/audio/selector/selector.c

```diff
@@ -579,14 +579,12 @@ static void build_config(struct comp_data *cd, struct module_config *cfg)
 	enum sof_ipc_frame __sparse_cache frame_fmt, valid_fmt;
 	int i;
 
-	frame_fmt = cfg->base_cfg.audio_fmt.depth;
 	audio_stream_fmt_conversion(cfg->base_cfg.audio_fmt.depth,
 				    cfg->base_cfg.audio_fmt.valid_bit_depth,
 				    &frame_fmt, &valid_fmt,
 				    cfg->base_cfg.audio_fmt.s_type);
 	cd->source_format = frame_fmt;
 
-	frame_fmt = cd->sink_format;
```

Comment:
Should this file be in a separate commit?

Reply:
Why? It is related only to removing xt-clang warning.

Reply:
Even if it's not a functional change, it very much looks like one.

Reply:
Let's focus on what it actually is and not what it looks to be. As some people say "don't judge the book by the cover" :-)

Reply:
@aborisovich You do need to explain  this in the git commit in some way why is it ok to remove these two lines. This is not unheard of to have code that doesn't do anything, but expectation is that you explain in commit to explain why this is the case.

Reply:
ok sure, I'll add some proper message